google/cloud/bigtable/examples/stock_backtesting/populate_data_main.cc

```diff
@@ -0,0 +1,219 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START examples_populate_data_main]
+
+// Parse the input csv file, and write the data into the BigTable.
```

Comment:
we do not typically leave comments above the includes. Please move somewhere else, or delete this.